src/dmd/parse.d

```diff
@@ -2028,12 +2028,14 @@ final class Parser(AST) : Lexer
         case TOK.functionString:
         case TOK.prettyFunction:
         case TOK.this_:
-            {
-                // Template argument is an expression
-                AST.Expression ea = parsePrimaryExp();
-                tiargs.push(ea);
-                break;
-            }
```

Comment:
This code block here was not respecting the norm that the other code blocks introduced, so I fixed the indentation.